src/main/scala/subsystem/BaseSubsystem.scala

```diff
@@ -77,6 +77,7 @@ abstract class BaseSubsystem(implicit p: Parameters) extends BareSubsystem with
   def getOMInterruptDevice(resourceBindingsMap: ResourceBindingsMap): Seq[OMInterrupt]
 
   val logicalTreeNode = new SubSystemLogicalTreeNode(getOMInterruptDevice)
+  LogicalModuleTree.add(LogicalModuleTree.rootLTN, logicalTreeNode)
```

Comment:
Should this really always be added to the root? What if there are other layers in between?